drf_spectacular/generators.py

```diff
@@ -102,6 +103,7 @@ def __init__(self, *args, **kwargs):
         self.registry = ComponentRegistry()
         self.api_version = kwargs.pop('api_version', None)
         self.inspector = None
+        self.webhooks = kwargs.pop('webhooks', None)
```

Comment:
not sure this actually adds something, given that we have the setting and we have a settings override mechanism for the view.